proctor-common/src/main/java/com/indeed/proctor/common/model/PayloadExperimentConfig.java

```diff
@@ -0,0 +1,44 @@
+package com.indeed.proctor.common.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.extern.jackson.Jacksonized;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Collections;
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Jacksonized
+public class PayloadExperimentConfig {
+    @Nullable private String priority;
+    @Nonnull private List<String> namespaces = Collections.emptyList();
+
+    @Override
+    public String toString() {
+        return "PayloadExperimentConfig{"
+                + "priority='"
+                + priority
+                + '\''
+                + ", namespaces="
+                + namespaces
+                + '}';
+    }
+
+    public static boolean isHigherPriority(
```

Comment:
if this is instance method, we can do thisPayloadExperimentConfig.isHigherPriorityThan(otherPayloadConfig).
Any reason we want it to be static?

Reply:
Moved to class method